validators/cgroup_validator_linux.go

```diff
@@ -55,32 +55,32 @@ func (c *CgroupsValidator) Validate(spec SysSpec) (warns, errs []error) {
 	var st unix.Statfs_t
 	var err error
 	if err := unix.Statfs(unifiedMountpoint, &st); err != nil {
-		return nil, []error{errors.Wrap(err, "cannot statfs the cgroupv2 root")}
+		return nil, []error{fmt.Errorf("cannot statfs the cgroupv2 root: %v", err)}
```

Comment:
How about using `: %w` here?
For example: https://go.dev/play/p/P9R4uGYZaqa

Reply:
Updated.